### DISCUSSION TITLE
Visualize MD trajectory in Pymol

### DISCUSSION THREAD

**Discussion by jacekkozuch** [TinkerTools/tinker-gpu#193] [Q&A] [answered]
Dear all,

I wanted to ask what is your favorite way to visualize your Tinker MD results in Pymol, since it doesn't like xyz and arc files. 

My current strategy is to translate the results into a pdb file and I'm sure that there are some nice tools for this already out there - or some alternative ways to deal with this.

Thx!
Jacek

**Comment by jayponder** [comment, marked as answer]
Some older versions of PyMOL do read Tinker files. But you are correct that reading Tinker files was removed at some point after Schrodinger took control of the code. As you note, if you are doing work on proteins or nucleic acids, then converting to PDB files is probably best. It has the advantage that you get the sequence information and can do fancy biomolecule-specific graphics representations.

An alternative is to use VMD to view Tinker files. VMD does read Tinker files directly. (There was a long standing bug in VMD's Tinker file reader, but this is now fixed in the most recent releases.) Although even with VMD, you may still want to convert to-from PDB files for viewing in order to get the sequence information.

The XYZPDB and PDBXYZ tools in Tinker are not perfect, but they are reasonably good as long as your structure does not contain unusual or modified residues.

**Reply by jacekkozuch** [reply]
Thanks for the quick response.
Unfortunately, I have unusual amino acids in my protein and therefore had issues with XYZPDB. 

I'll see about VMD. Thanks for the suggestion!
